Run-SystemTest.ps1

```diff
@@ -0,0 +1,37 @@
+#Requires -Version 5.0
+
+Begin {
+    $ErrorActionPreference = "stop"
+}
+
+Process {
+    function Exec([scriptblock]$Command) {
+        & $Command
+        if ($LASTEXITCODE -ne 0) {
+            throw ("An error occurred while executing command: {0}" -f $Command)
+        }
+    }
+    
+    $workingDir = Join-Path $PSScriptRoot "src"
+    $outputDir = Join-Path $PSScriptRoot ".output"
+    $sysTestDir = Join-Path $PSScriptRoot "src/system-test/WebApi.MsBuild.SystemTest"
+
+    try {
+        Push-Location $workingDir
+        Remove-Item $outputDir -Force -Recurse -ErrorAction SilentlyContinue
+    
+        Exec { & dotnet pack -c Release -o "$outputDir" }
+
+        Push-Location $sysTestDir
+
+        Exec { & dotnet nuget locals all --clear }
+        Exec { & echo "<?xml version=`"1.0`" encoding=`"utf-8`"?><configuration><packageSources><add key=`"local-packages`" value=`"$outputDir`" /></packageSources></configuration>" > NuGet.Config }
```

Comment:
Oh damn fancy pass pass! Maybe we could explain it in the Readme for futur maintainer.

Reply:
Is nugget installed on the runner. If yes maybe we could use their CLI to setup the reference instead of relying on it's implementation?  (https://spin.atomicobject.com/2021/01/05/local-nuget-package/)
![image](https://github.com/gsoft-inc/wl-openapi-msbuild/assets/8277266/adc31fd7-3458-4af3-973a-e3c92860a256)

Reply:
Maybe it could even be possible to avoid any file manipulation with this approach:

https://weblog.west-wind.com/posts/2022/Sep/11/Referencing-a-Local-Private-NuGet-Package-in-your-Solution

![image](https://github.com/gsoft-inc/wl-openapi-msbuild/assets/8277266/fcc7bbfd-0696-40f7-a5c0-4557c5550775)

Reply:
Otherwise simply having SystemTest referencing the MSBuild project is not enough?

Reply:
The purpose of this test is to verify if our MSBuild task nuget can be installed on a `clean` project and run out of the box without any explicit reference to our MSBuild Project.